chapters/infrastructure/processing/reading/README.md

```diff
@@ -1,12 +1,41 @@
+---
+title: Content Processing
+---
+
 ### Content Processing
 
-Owner: Vlad Năstase
-Reviewer: Teo Duțu
+Content organized in a Version Control System like `git`, and published to a `GitHub` repository is suitable for easier contributions.
+In order to deliver it to the end user/learner, this content has to be processed and rendered in an easier to navigate form.
+
+The types of content that need to be processed vary with course requirements and require different actions in order to be rendered.
+These types include, but are not limited to:
+
+- slides
+- reading material
+- quizes
+- various types of media content.
+
+The rendered output that we recommend is a website generated using a static website generator.
+The usage of [Docusaurus](https://docusaurus.io/) is also recommended, but other static website generators (Hugo, Docsy, Sphinx) can be used.
+The rendered website can embed materials such as slides and quizes, and can also refer the end user/learner to the `GitHub` repository from which the content is generated (or to an auxiliary repository) for practical content, such as drills.
+
+Most static website generators have a documented structure that needs to be followed for the website to render.
+This can be incompatible with the way content is organized for easier collaboration.
```

Comment:
This is a bit unclear. You're saying that the content should be structured in such a way as to fit the publisher?